src/js/gameController.js

```diff
@@ -38,7 +39,8 @@ var gameController = {
     hasReachedCombo: false,
     regularEggPoints: 5,
     scoreBoostPoints: 30,
-    frenzyPoints: 20,
+    baseFrenzyPoints: 5,
+    frenzyPoints: 5,
```

Comment:
I don’t understand from the name what the difference is between these variables…?